py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -7,32 +7,34 @@
 from collections.abc import Iterable
 from typing import NamedTuple
 
-from py_gtfs_rt_ingestion import batch_files, file_list_from_s3
+from py_gtfs_rt_ingestion import batch_files
+from py_gtfs_rt_ingestion import file_list_from_s3
 
 import logging
 logging.basicConfig(level=logging.INFO)
 
 DESCRIPTION = "Generate batches of json files that should be processed"
 
 class BatchArgs(NamedTuple):
-    input_file: str
     s3_prefix: str
+    s3_bucket: str
     filesize_threshold: int
-    output: str
     pretty: bool
 
-def parseArgs(args) -> BatchArgs:
+def parseArgs(args) -> dict:
```

Comment:
return an event dict. pipe it directly into `lambda_handler`

same adjustments as mentioned above.